simulation.py

```diff
@@ -277,23 +277,26 @@ def epsilon_to_png(self):
                     h5_file=self.eps_file + ':data', 
                     output_file=defaults.temporary_epsh5)
         log.info("calling: {0}".format(callstr))
-        if not sp.call(callstr.split(), cwd=self.workingdir):
-            # no error, continue:
-            dct = dict(self.__dict__, h5_file=defaults.temporary_epsh5)
-            # save dielectric to png:
-            if self.geometry.is3D:
-                callstr = (defaults.epsh5topng_call_3D % dct,
-                           defaults.epsh5topng_call_3D_cross_sect % dct)
-            else:
-                callstr = (defaults.epsh5topng_call_2D % dct,)
-            retcode = 0
-            for s in callstr:
-                if not retcode:
-                    log.info("calling: {0}".format(s))
-                    retcode = retcode or sp.call(s.split(), 
-                                                 cwd=self.workingdir)
-        else:
+        
+        try:
+            sp.call(callstr.split(), cwd=self.workingdir)
+        except:
```

Comment:
Could you add the exact Exception you are catching here? It's not good to catch all possible errors - it could hide some new error in future.